components/allScores.tsx

```diff
@@ -60,7 +60,21 @@ const newCols = [
 ];
 
 export const exportCSV: any = (work: any) => {
-	const csvExporter = new ExportToCsv();
+	const csvExporter = new ExportToCsv({
+		filename: 'judging-data',
+		showLabels: true,
+		headers: [
+			'Technical Ability',
+			'Creativity',
+			'Utility',
+			'Presentation',
+			'Wow Factor',
+			'Comments',
+			'Feedback',
+			'Team',
+			'Judge',
```

Comment:
This same list is duplicated in some other places too. I think it would be nice to have these be a constant somewhere. What do you think?

Reply:
I agree, otherwise we will have to maintain the list of fields in 2 places, ideally there is a programmatic way to get these from our models or so

Reply:
yeah looks like you can just make it from newCols at least